README.md

```diff
@@ -6,9 +6,9 @@ Alongside GLib bindings, this crate also includes the infrastructure to
 subclass `GObject` for extending libraries like GTK+. A code example can
 be found in the documentation of this crate in the `subclass` module.
 
-- [Gtk-rs project site](http://gtk-rs.org/)
+- [Gtk-rs project site](https://gtk-rs.org/)
 
-- [Online documentation](http://gtk-rs.org/docs/)
+- [Online documentation](https://gtk-rs.org/docs/)
```

Comment:
Actually, the good link is this one: https://gtk-rs.org/docs-src/

Reply:
Ah, there are a lot of links pointing to the old folder

Reply:
@GuillaumeGomez Can we maybe have `docs` forward to `docs-src`? Lots of search results with Google also point to the wrong location.

Reply:
It's worth a try at least.

Reply:
Apparently, from what I can understand, it requires access to wherever the domain name was registered to be able to add a new redirection. So it's up to @gkoz, I can't do anything about it. :-/